src/components/Common/AvatarEditModal.tsx

```diff
@@ -145,9 +267,105 @@ const AvatarEditModal = ({
       toast.warning(t("please_upload_an_image_file"));
       return;
     }
+
+    // Check file size (1MB = 1048576 bytes)
+    if (file.size > 1048576) {
```

Comment:
never hard-code such things directly inside the code!

always keep constants referenced from a global variable or from env/careConfig.

what if we ever want to change these limits in the future? 🤔 

write maintainable code!